src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +74,63 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(event) => (
+                // eslint-disable-next-line no-sequences
+                handleQueryChange(event.target.value),
+                setCatchError(false)
+              )}
```

Comment:
move it to a separate function
do not disable lint, try to figure out problem (google/chat)